includes/base/controls-stack.php

```diff
@@ -305,7 +305,7 @@ public function get_controls( $control_id = null ) {
 	 * @return array Active controls.
 	 */
 	public function get_active_controls( array $controls = null, array $settings = null ) {
-		// _deprecated_function( __METHOD__, '3.0.0' );
+		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( __METHOD__, '3.0.0' );
```

Comment:
@rami-elementor from which version it counts as deprecated?

Reply:
If it was deprecated in 3.0, the code should be deleted in 3.8.

Reply:
What should I do here? can I resolve this conversion?

Reply:
@davseve your change is exactly what should be done.